<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-V7AFDP1" failures="1" tests="3" name="cucumberRunner.CucumberTestNGRunner" time="72.303" errors="0" timestamp="2021-12-29T15:30:54 IST" skipped="0">
  <testcase classname="cucumberRunner.CucumberTestNGRunner" name="runScenario" time="14.575">
    <failure message="expected [My account - My Store] but found [508 Resource Limit Is Reached]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [My account - My Store] but found [508 Resource Limit Is Reached]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at cucumberPackage.loginTest.check_for_the_successful_login(loginTest.java:51)
at âœ½.check for the successful login(file:///C:/Users/Veeru%20%20Sai/Desktop/AutomationProject/src/test/resources/Features/loginAuth.feature:20)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="cucumberRunner.CucumberTestNGRunner" name="runScenario" time="31.599"/>
  <system-out/>
  <testcase classname="cucumberRunner.CucumberTestNGRunner" name="runScenario" time="26.129"/>
  <system-out/>
</testsuite> <!-- cucumberRunner.CucumberTestNGRunner -->
